arch/risc-v/src/espressif/Kconfig

```diff
@@ -213,6 +213,10 @@ config ESPRESSIF_UART1
 	select UART1_SERIALDRIVER
 	select ARCH_HAVE_SERIAL_TERMIOS
 
+config ESPRESSIF_HR_TIMER
+	bool "High Resolution Timer"
+	default n
```

Comment:
Please include a --help-- to explain what it the HR_TIMER, maybe also comment it is the equivalent to RT_TIMER on ESP32/Sx

Reply:
Done!